tests/local_db/test_data_access.py

```diff
@@ -218,7 +218,9 @@ def test_group_comment_delete_bad_params():
         comment="author comment",
     )
     with pytest.raises(exceptions.APIException):
-        dal.group_comment_delete(release_request.id, "badgroup", "1", author, audit)
+        dal.group_comment_delete(
+            release_request.id, "badgroup", "1", author.username, audit
+        )
```

Comment:
this DAL method should be called with a string, but was being called with a BLL User object